docs/topics/vscode-extension-configuring-analysis.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-configuring-analysis_{context}']
+= Configuring a run configuration
+
+You can configure multiple run configurations to run against each project.
+
+.Prerequisites
+
+* Projects imported into VS Code.
```

Comment:
I'm still wondering about this prerequisite. Something tells me that it is too obvious. I would not expect to use a VS Code extension to analyze a project that does not exist in VS Code, so I am wondering why we need to tell the user to do this.

Reply:
Rewrote the introductory sentence to include the requirement.